src/amp-story-ad-preview.js

```diff
@@ -62,8 +62,13 @@ const httpsCircumventionPatch = minifyInlineJs(`
   })();
 `);
 
-const insertHttpsCircumventionPatch = docStr =>
-  docStr.replace('<head>', `<head><script>${httpsCircumventionPatch}</script>`);
+const insertPatches = docStr => {
```

Comment:
Let's keep `insertHttpsCircumventionPatch`, add a `setBodyAmpStoryVisible`, and use both in `insertPatches`.